README.md

```diff
@@ -7,38 +7,61 @@
 [![contributers](https://img.shields.io/github/contributors/chichou/grapefruit)](https://github.com/ChiChou/Grapefruit/graphs/contributors)
 [![License](https://img.shields.io/github/license/chichou/grapefruit)](https://github.com/ChiChou/Grapefruit/blob/master/LICENSE)
 
+
 ## Get Started
+ 
+Grapefruit requires the following tools to build:
+
+1. [Node.js][node] `LTS`. For Apple Silicon, use Node.js `Current`.
+2. [Frida][frida] on computer.
+3. For jailbroken iOS [Frida iOS][frida ios].
+3. Tmux or Windows Terminal on Windows.
 
 ![Screenshot](images/screenshot.png)
 
-Git clone:
 
-`git clone --recurse-submodules https://github.com/ChiChou/Grapefruit`
+## Installing Grapefruit
```

Comment:
Also this project is under development. I wouldn't call it installing because the installation should be much more easier after it's ready

Reply:
Maybe Build Grapefruit?

Reply:
@ChiChou ^